### PR TITLE
Update fuchsia.toml to reflect team changes

### PR DESCRIPTION
**Pull request by tmandry**
cc @erickt

### CODE REVIEW

File: teams/fuchsia.toml [hunk 1]
```diff
@@ -6,6 +6,6 @@ leads = []
 members = [
     "tmandry",
     "djkoloski",
-    "ComputerDruid",
+    "erickt",
```
Comment (WaffleLapkin) [inline teams/fuchsia.toml:9]: Doesn't this require changing files in `people/` too?

Comment (rylev) [reply teams/fuchsia.toml:9]: Indeed @tmandry CI is failing.

**Comment by jackh726** [comment]
ping @tmandry can you fix CI?

**Comment by dtolnay** [comment]
> <code>[ERROR rust_team::validate] validation error: person `ComputerDruid` is not a member of any team (active or archived), has no permissions, and is not an individual contributor to any repo</code>

Please add:

```toml
alumni = [
    "ComputerDruid",
]
```

to the bottom of this file, or delete people/ComputerDruid.toml.

> <code>[ERROR rust_team::validate] validation error: alumni team includes active member `erickt`</code>
> <code>[ERROR rust_team::validate] validation error: user `jntrnr` changed username to `sophiajt`</code>

These will go away as soon as you rebase past #1117 + #1126.

**Review by rylev — APPROVED** [review]

**Pull request merged**